firmware/common/ak4951.hpp

```diff
@@ -837,6 +837,10 @@ class AK4951 : public audio::Codec {
     void speaker_enable();
     void speaker_disable();
 
+    bool speaker_disable_supported() const override {
```

Comment:
Not code related: Which device is has which audio chip? I don't know which one is which.

Reply:
This Wiki page has some good info on which PortaPack models have AK4951 vs WM8731:  https://github.com/eried/portapack-mayhem/wiki/PortaPack-Versions